kedro-datasets/requirements.txt

```diff
@@ -1 +1 @@
-kedro
+kedro==0.18.4
```

Comment:
not sure if this is required?

Reply:
I think it can be `kedro~=0.18.4` as @noklam wrote in https://github.com/kedro-org/kedro/issues/2004

Reply:
done!